client/src/analytics/events.ts

```diff
@@ -50,7 +50,8 @@ export enum EVENTS {
   EXPLORER_GENE_INFO_BUTTON_CLICKED = "EXPLORER_GENE_INFO_BUTTON_CLICKED",
   WMG_CLICK_NAV = "WMG_CLICK_NAV",
   COLLECTIONS_CLICK_NAV = "COLLECTIONS_CLICK_NAV",
-  CENSUS_CLICK_NAV = "CENSUS_CLICK_NAV",
```

Comment:
@seve is @ainfeld aware of the removal of this event? Thank you!

Reply:
Yep! This was my request, thanks @tihuan for checking 🙏

Reply:
Awesome thanks for confirming!